package.json

```diff
@@ -46,13 +48,13 @@
     "symlink": "ln -s $PWD/bin/index /usr/local/bin/fly",
     "clean": "rimraf dist",
     "lint": "eslint src test",
-    "coverage": "babel-node node_modules/.bin/isparta cover test/*.js",
+    "coverage": "babel-node --presets es2015,stage-3 ./node_modules/.bin/isparta cover test/*.js",
```

Comment:
@marvinroger @bucaran take a look at latest change in [tiny-es-nm generator](https://github.com/iamstarkov/generator-tiny-es-nm) i finally got coverage working with babel v6. It’s here https://github.com/iamstarkov/generator-tiny-es-nm/blob/master/app/templates/_package.json#L8